startup/browser/ocio.py

```diff
@@ -0,0 +1 @@
+../../startup/gui/ocio.py
```

Comment:
(a) I had no idea this would work

(b) should we consider making the ocio config a module level config? I seem to recall it being a sore point at some time in the past that we needed to replicate it out of gui (though I don't remember how/when that came up).

Reply:
It can't be a module level config because it needs access to an application. There's probably a case to be made for moving the functionality to IECoreImage and then calling it from the config so that it can be shared more easily, but at this point I'm just aiming to do the minimum to keep us where we were at before the IECoreImage change.